### DISCUSSION TITLE
https://theme-hope.vuejs.press/zh/config/theme/basic.html

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#796] [Announcements]
# https://theme-hope.vuejs.press/zh/config/theme/basic.html

警告 这些选项很重要，需要你正确配置。 hostname 类型: string 必填: 是 当前网站部署到的域名。 提示 它应该包含完整协议 (如: https://example.com)。 author 类型: Author 必填: 否 文章显示的默认作者 license 类型: string 必填: 否 站点的默认协议 favicon 类型: s...

https://theme-hope.vuejs.press/zh/config/theme/basic.html

**Comment by brycenhe** [comment]
Hi，请问下这个hostname字段有什么实际作用？

**Reply by Mister-Hope** [reply]
你可以自己翻源码

**Reply by muchfish** [reply]
建议不要回答。免得说的都是废话

**Comment by izhaong** [comment]
真不错
